python/src/scipp/html/formatting_html.py

```diff
@@ -171,6 +170,23 @@ def summarize_mask(dim, var, ds=None):
     return summarize_variable(str(dim), var, is_index=False, embedded_in=ds)
 
 
+def find_bin_edges(var, ds):
+    """
+    Checks if the coordinate contains bin-edges.
```

Comment:
Function name is good, but comment implies we get a bool return.

Reply:
I didn't change this, this is part of the `git revert` of https://github.com/scipp/scipp/pull/2086.